components/TextArea.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+const TextArea = ({
+  label,
+  name,
+  onChange,
+  required,
+  minLength,
+  maxLength,
+  width,
+  height,
+  padding,
+  margin,
+}) => (
+  <div className="input-root">
+    <label htmlFor={name}>{label}</label>
```

Comment:
A easier way is to wrap <input> in the <label><input/></label> and this way you can avoid the for=

Reply:
did what you suggested, work like charm